src/components/Synthetics/TradeBox/NewCollateralSelector/NewCollateralSelector.tsx

```diff
@@ -0,0 +1,190 @@
+/* eslint-disable react/no-unused-prop-types */
```

Comment:
removed this disabling from the beginning of the file
but there are still three places where it had to be done, ikd why eslint does not pick this up but i think its because i pass props as spread